spec/barbeque/executor/hako_spec.rb

```diff
@@ -418,6 +454,48 @@
           end
         end
       end
+
+      context 'when retried job fails for retry limit of times' do
```

Comment:
This context can be placed in the "when retried job fails" context (just like docker_spec.rb)

Reply:
Exactly... fixed https://github.com/cookpad/barbeque/pull/77/commits/53a56fbad19e7daca3544811305fd883950e8035